src/community/geogig/src/main/java/org/geogig/geoserver/gwc/MinimalDiffBoundsConsumer.java

```diff
@@ -89,7 +89,7 @@ private void addEnv(@Nullable Bounded node) {
         if (node == null) {
             return;
         }
-        final Envelope env = node.bounds().orNull();
+        final Envelope env = node.bounds().get();
```

Comment:
This will now throw an exception if the bounds are null. A more equivalent statement would be 
```
final Envelope env = node.bounds().orElse(null);
```